Bottom: 38315790561b251448be936be2cc4e49b47e572a
Top:    ddcb9a923a66d9ad09b29eb5d98c6359f541fc62
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:21:04 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 49462ca..7260fa5 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -290,7 +290,7 @@ get_contour_coord_n_in_mm (PLINE *contour, int n, double *x, double *y)
 
 
 void
-object3d_export_to_step (object3d *object, char *filename)
+object3d_export_to_step (object3d *object, const char *filename)
 {
   FILE *f;
   time_t currenttime;
diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index 8d081a2..9a34fa8 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -17,5 +17,5 @@ void object3d_add_vertex (object3d *object, vertex3d *vertex);
 void object3d_add_face (object3d *object, face3d *face);
 object3d *object3d_create_test_cube (void);
 object3d *object3d_from_board_outline (void);
-void object3d_export_to_step (object3d *object, char *filename);
+void object3d_export_to_step (object3d *object, const char *filename);
 void object3d_test_board_outline (void);
